src/server/report/report.service.ts

```diff
@@ -82,6 +83,7 @@ export class ReportService {
         isSEAS: isSEASEnumToString(course.isSEAS),
         isUndergrad: course.isUndergraduate ? 'Undergraduate' : 'Graduate',
         notes: course.notes,
+        sameAs: course.sameAs,
```

Comment:
The ticket suggests possibly bolding or putting brackets around the parent course. Was this discussed further?